core/domain/suggestion_services.py

```diff
@@ -44,8 +44,8 @@ def create_suggestion(
         suggestion_type: str. The type of the suggestion.
         target_type: str. The target entity being edited.
 
-        (The above 2 parameters should be one of the constants defined in
-        storage/suggestion/gae_models.py.)
+            (The above 2 parameters should be one of the constants defined in
```

Comment:
Drop this, and add to the end of each of the arg descriptions above: "This parameter should ..."

Reply:
Done